packages/client/src/Components/MessageCreator/MessageCreator.tsx

```diff
@@ -82,7 +85,129 @@ const MessageCreator: React.FC<Props> = (props) => {
     }
   }, [props])
 
+  /**
+   * helper function to render default Datetime or Date or Time props of json
+   */
+  const configCommonProps = (prop: any) => {
+    switch (prop.format) {
+      case "datetime-local":
+        prop.default = moment(gameDate).format("DD/MM/YYYY HH:mm")
+        prop.options !== undefined ? prop.options.flatpickr = flatpickr(".calendar") : prop.options = {}
+        prop.options = {"flatpickr": {
+          "wrap":false,
+          "time_24hr": true,
+          "dateFormat":"d/m/Y H:i",
+        }}
+        return prop
+      case "date":
+        prop.default = moment(gameDate).format("DD/MM/YYYY")
+        prop.options !== undefined ? prop.options.flatpickr = flatpickr(".calendar") : prop.options = {}
+        
+        prop.options = {"flatpickr": {
+          "wrap":false,
+          "dateFormat":"d/m/Y",
+        }}
+        return prop
+      case "time":
+        prop.default = moment(gameDate).format("HH:mm")
+        prop.options !== undefined ? prop.options.flatpickr = flatpickr(".calendar") : prop.options = {}
+        
+        prop.options = {"flatpickr": {
+          "wrap":false,
+          "time_24hr": true,
+          "dateFormat":"H:i",
+        }}
+        return prop
+      default:
+        return prop
+    }
+  }
+
+  /**
+   * Render Default datetime entries in template of json for type datetime-local
+   */
+  const configDateTimeLocal = (schema: any) => {
+    if(!schema || !schema.properties){
+      return
+    }
+    Object.keys(schema.properties).forEach(key => {
+      let prop = schema.properties[key]
+      if(prop.format === 'datetime-local' || prop.format === 'date' || prop.format === 'time'){
```

Comment:
@sanjayradadiya  - we to string comparison above, then we do it again in the switch block in `configCommonProps `

I'm pretty sure we can remove this `if` construct, can't we?

Reply:
Yes, you are right.
I have two options at that time.
1) function should be called every time irrespective of format
2) function should be called only if we have format datetime or date or time
I choose option 2 and I will change it to option 1.

Reply:
option 2 changed to option 1 in latest commit.